roles/openshift_examples/files/examples/v3.6/cfme-templates/jboss-middleware-manager-template.yaml

```diff
@@ -175,6 +190,9 @@ objects:
         - name: h-services-data
           persistentVolumeClaim:
             claimName: h-services-pvc
+        - name: hawkular-secret-volume
+          secret:
+            secretName: hawkular-ssl-secret
```

Comment:
Will there be the SSL solution with secrets, or there should be Routes solution?

Reply:
It's a routes solution, but even that solution needs for mount some secret, in this case is the secret created by openshift signed service.